### ISSUE TITLE
Mutiple NFT Art Pieces in one NFT collection

### ISSUE THREAD

**Issue by gersiaan** [HashLips/hashlips_art_engine#1151]
I have several art pieces, which differs in the art itself and has different layers to them. Is it possible to create several different artworks and create them in one collection?

**Comment by bolshoytoster** [comment]
@gersiaan yes, you can have multiple configs in your `layerConfigurations`, e.g.
```js
const layerConfigurations = [
  {
    // First artwork
    growEditionSizeTo: 5,
    layersOrder: [
      { name: "Background" },
      { name: "Eyeball" },
      { name: "Eye color" },
      { name: "Iris" },
      { name: "Shine" },
      { name: "Bottom lid" },
      { name: "Top lid" },
    ],
  },
  {
    // Second artwork
    growEditionSizeTo: 5,
    layersOrder: [
      { name: "Background" },
      { name: "Eyeball" },
      { name: "Eye color" },
      { name: "Iris" },
      { name: "Shine" },
      { name: "Bottom lid" },
      { name: "Top lid" },
    ],
  },
];
```

**Comment by samimikawi** [comment]
hello i checked it but it keep on generate the first one and nothing and not continue

**Comment by samimikawi** [comment]
const layerConfigurations = [
  {
    // First artwork
    growEditionSizeTo: 5,
    layersOrder: [
      { name: "Background" },
      { name: "Body" },
      { name: "Left Arm Ballon" },
      { name: "Right Arm" },
      { name: "Neck" },
      { name: "Head" },
      { name: "Cut" },
      { name: "Facial" },
      { name: "Hair" },
      { name: "Ballons Left" },
    ],
  },
  {
    // Second artwork
    growEditionSizeTo: 5,
    layersOrder: [
      { name: "Background" },
      { name: "Body" },
      { name: "Left Arm" },
      { name: "Right Arm Ballon" },
      { name: "Neck" },
      { name: "Head" },
      { name: "Cut" },
      { name: "Facial" },
      { name: "Hair" },
      { name: "Ballons Right" },
    ],
  },
];

**Comment by bolshoytoster** [comment]
@samimikawi make sure you save the file after you make changes.

**Comment by samimikawi** [comment]
yes i did and i close the entire VB and open it again and try again only run the first one the dosen't continue

**Comment by bolshoytoster** [comment]
@samimikawi could you try using [my fork](https://github.com/bolshoytoster/hashlips_art_engine) please.

**Comment by samimikawi** [comment]
Works Thanks
